team.html

```diff
@@ -0,0 +1,333 @@
+<li>Dirk Lennarz (<a href="https://github.com/dlennarz">@dlennarz</a>), Michigan Univ., US</li>
+<li>Domenico Tiziani (<a href="https://github.com/dltiziani">@dltiziani</a>), Erlangen Univ., Germany</li>
+<li>Ellis Owen (<a href="https://github.com/ellisowen">@ellisowen</a>), UCL, US</li>
+<!--<li>Emma de Oña-Wilhelmi (), IEEC, Spain</li>
+<li>Enrique Mestre Guillén (), IEEC, Spain</li>
+-->
+<li>Fabio Acero (<a href="https://github.com/facero">@facero</a>), SAP/CEA, France</li>
+<li>Helen Poon (<a href="https://github.com/helen-poon">@helen-poon</a>), MPIK, Germany</li>
+<li>Ignasi Reichardt (<a href="https://github.com/ignasi-reichardt>">@ignasi-reichardt</a>), APC, France<sup>*</sup></li>
+<li>Johannes King (<a href="https://github.com/joleroi">@joleroi</a>), MPIK, Germany</li>
+<li>Jonathan Harris (<a href="https://github.com/JonathanDHarris">@JonathanDHarris</a>), JON, UK</li>
+<li>José Enrique Ruiz (<a href="https://github.com/Bultako">@Bultako</a>), IAA, Spain</li>
+<li>Julien Lefaucheur (<a href="https://github.com/jjlk">@jjlk</a>), SAP/CEA, France</li>
+<li>Lars Mohrmann (<a href="https://github.com/lmohrmann">@lmohrmann</a>), Erlangen Univ., Germany</li>
+<li>Léa Jouvin (<a href="https://github.com/JouvinLea">@JouvinLea</a>), IFAE, Spain</li>
+<li>Luigi Tibaldo (<a href="https://github.com/tibaldo">@tibaldo</a>), IRAP, France</li>
+<li>Manuel Paz Arribas (<a href="https://github.com/mapazarr">@mapazarr</a>), Humboldt Univ., Germany<sup>*</sup></li>
+<li>Matthew Wood (<a href="https://github.com/woodmd">@woodmd</a>), SLAC, US</li>
+<li>Matthias Wegen (<a href="https://github.com/wegenmat">@wegenmat</a>), DESY Zeuthen, Germany</li>
+<li>Nachiketa Chakraborty (<a href="https://github.com/cnachi">@cnachi</a>), MPIK, Germany</li>
+<li>Olga Vorokh (<a href="https://github.com/OlgaVorokh">@OlgaVorokh</a>),, Belarus</li>
+<li>Peter Deiml (<a href="https://github.com/pdeiml">@pdeiml</a>), Erlangen Univ, Germany</li>
+<li>Régis Terrier (<a href="https://github.com/registerrier">@registerrier</a>), APC, France</li>
+<li>Roberta Zanin (<a href="https://github.com/robertazanin">@robertazanin</a>), MPIK, Germany</li>
+<li>Rolf Bühler (<a href="https://github.com/rbuehler">@rbuehler</a>), DESY Zeuthen, Germany</li>
+<li>Rubén López-Coto (<a href="https://github.com/rlopezcoto">@rlopezcoto</a>), Padova Univ, Italy</li>
+<li>Stefan Klepser (<a href="https://github.com/klepser">@klepser</a>), DESY Zeuthen, Germany</li>
+<li>Thomas Armstrong (<a href="https://github.com/thomasarmstrong">@thomasarmstrong</a>), Manchester Univ., UK</li>
+<li>Thomas Vuillaume (<a href="https://github.com/vuillaut">@vuillaut</a>), LAPP, France</li>
+<li>Victor Zabalza (<a href="https://github.com/zblz">@zblz</a>), Leads Univ, UK<sup>*</sup></li>
+<li>Zé Vinícius (<a href="https://github.com/mirca">@mirca</a>), NASA, US</li>
+    </ul>
+<p><sup>*</sup>: Has quit the project.</p>
```

Comment:
I'm suggesting to remove the "has quit the project". It's almost impossible to draw that line and tedious to maintain, we would need guidelines like in H.E.S.S. how long people stay in the "active" status. And with Gammapy the hope would always be that people come back to Gammapy at some point, e.g. a PhD that does some development, then does something else for a year or two, and maybe comes back and does some Gammapy development as a Postdoc.

I've worked on this team page today and will put up an updated proposal soon. I just wanted to mention this point now, why (to be discussed) I'm removing the "has quit the project" markers.